rusoto/services/sts/src/custom/web_identity.rs

```diff
@@ -30,23 +30,21 @@ pub struct WebIdentityProvider {
     /// associated with the user who is using your application. That way, the temporary security credentials
     /// that your application will use are associated with that user. This session name is included as part
     /// of the ARN and assumed role ID in the AssumedRoleUser response element.
-    pub role_session_name: Variable<String, CredentialsError>,
+    pub role_session_name: Variable<Option<String>, CredentialsError>,
```

Comment:
Now, one has to always pass a Variable for `role_session_name`, while with the former approach one could provide `None` here. I propose we change this to:
```rust
pub role_session_name: Option<Variable<Option<String>, CredentialsError>>,
```
The idea is that if a user of those API does not care about the session name, he should have to provide a `Variable` at all, so we must allow him to provide a `None` value instead.